docker.local/config/sc.yaml

```diff
@@ -202,6 +202,14 @@ smart_contracts:
       trigger_period: 30
       blobber_capacity_ratio: 10
       blobber_usage_ratio: 40
+      gamma:
```

Comment:
change the values to following:
A: 10
B: 9
alpha: 0.2
i: 1
K: 0.9
mu: 0.2

we need to get these changes in helm as well.

Reply:
Done